bundles/org.openhab.binding.ojelectronics/src/main/java/org/openhab/binding/ojelectronics/internal/models/groups/Schedule.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.ojelectronics.internal.models.groups;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * Model for a schedule
+ *
+ * @author Christian Kittel - Initial contribution
+ */
+@NonNullByDefault
```

Comment:
I think it might be dangerous to annotate your JSON classes like this, it gives a false sense of security. GSON will not validate the `null`-ness of the fields. And I don't know whether your bases it on some documentation but even that is not guaranteed to remain as is.

@cpmeister what do you think of this?

Reply:
If i do so, i think i got a warning.

Reply:
> I think it might be dangerous to annotate your JSON classes like this, it gives a false sense of security. GSON will not validate the null-ness of the fields. And I don't know whether your bases it on some documentation but even that is not guaranteed to remain as is.

@openhab/add-ons-maintainers your opinions are greatly appreciated here :-)

Reply:
@EvilPingu if no one veto's this one tonight and you looked into the latest remarks it should be ready to be merged! Thanks for your quick responses.